pages/recipe/index.js

```diff
@@ -785,13 +796,9 @@ class RecipePage extends React.Component {
               >
                 <button
                   className="btn btn-default"
-                  id="cmpsr-btn-crt-compos"
-                  data-toggle="modal"
-                  data-target="#cmpsr-modal-crt-compos"
                   type="button"
-                >
-                  Create Composition
-                </button>
+                  onClick={(e) => this.handleShowModalCreateComp(e)}
```

Comment:
Should do the same change for Create Composition button on Details, Components, and Revisions tab page.